Header.php

```diff
@@ -27,10 +27,11 @@
 <header>
  
    
-<link href="css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
+<link href="http://118.25.96.118/nor/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
 
 <!--<link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css"/> -->
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/fontawesome.min.css" />
+<!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/fontawesome.min.css" />-->
+<link rel="stylesheet" href="./webfonts/all.min.css" />
```

Comment:
@dzr201732120115 

You created a new folder called `webfonts`?

Is `all.min.css `the same as `fontawesome.min.css` ?


Hui